packages/vite/src/node/build.ts

```diff
@@ -225,13 +203,7 @@ export interface LibraryOptions {
 
 export type LibraryFormats = 'es' | 'cjs' | 'umd' | 'iife'
 
-export type ResolvedBuildOptions = Required<
-  Omit<
-    BuildOptions,
-    // make deprecated options optional
-    'base' | 'cleanCssOptions' | 'polyfillDynamicImport' | 'brotliSize'
```

Comment:
So do we have the same situation here? It's using `Omit`

Reply:
We don't. They are already deprecated options from Vite.